conf/messages

```diff
@@ -154,13 +154,16 @@ chargeTypeDetails.insetTextReferenceAgent = Your client must use the 14-characte
 chargeTypeDetails.penaltiesAndAppeal = View your VAT penalties and appeals
 chargeTypeDetails.penaltiesAndAppealAgent = View your client’s VAT penalties and appeals
 
-estimatedInterest.vatChargeInterest = We charge interest on any unpaid VAT.
-estimatedInterest.penaltyChargeInterest = We charge interest on any unpaid late payment penalties.
+estimatedInterest.vatChargeInterest = We charge late payment interest on any unpaid VAT.
+estimatedInterest.penaltyChargeInterest = We charge late payment interest on any unpaid penalties.
 estimatedInterest.vatIncreaseDaily = The total increases daily based on the amount of unpaid VAT for the period.
 estimatedInterest.penaltyIncreaseDaily = The total increases daily based on the amount of unpaid penalty for the period.
 estimatedInterest.calculation = The calculation we use for each day is:
-estimatedInterest.vatCalculation = (Interest rate of {0}% × VAT amount unpaid) ÷ days in a year
-estimatedInterest.penaltyCalculation = (Interest rate of {0}% × penalty amount unpaid) ÷ days in a year
+estimatedInterest.vatCalculation = (Interest rate × VAT amount unpaid) ÷ days in a year
+estimatedInterest.currentRate = The current interest rate is {0}%.
+estimatedInterest.rateChanges = If the interest rate changes during the time interest is building up, we use the old interest rate up to the change date, then the new one after that. You can
+estimatedInterest.prevIntRateLink = find previous interest rates on GOV.UK (opens in a new tab)
```

Comment:
In the mural there seems to be a full stop at the end, after (opens in a new tab)